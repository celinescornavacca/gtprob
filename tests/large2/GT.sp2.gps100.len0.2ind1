((((((A8,A9),(((A41,A42),A4),((((A18,(A30,A31)),((A88,A89),A26)),((A82,A83),A55)),A13))),((A29,(A33,A34)),A11)),((A49,A50),A2)),(((((((A92,A93),A71),A65),((A39,A40),((A96,A97),(A35,(A48,(A66,(A73,A74))))))),(A5,(((A27,A28),A15),A10))),(A0,((A19,A20),(A12,(A94,A95))))),(((((A43,(A69,A70)),A25),(((A86,A87),A56),(A98,A99))),((((A3,((A14,((((A46,A47),(A53,A54)),A32),(A60,A61))),(A57,A58))),(A36,A37)),(A21,A22)),(((A44,(A77,A78)),((A45,((A84,A85),A81)),(A62,(A67,A68)))),((((A23,A24),(((A16,A17),A7),(A51,A52))),A1),((((A75,A76),A72),(A79,A80)),(A6,((A59,(A63,A64)),A38))))))),(A90,A91)))),((((B3,B4),B0),(((B22,(B92,B93)),((B52,B53),B14)),(((((B94,B95),B91),B36),(B45,B46)),(((B96,B97),B73),(B63,B64))))),((((B83,B84),B82),((B39,B40),(((((B25,(B74,B75)),B17),((B11,B12),B9)),B6),B5))),((((B1,((B7,(((B26,B27),((B60,B61),((B87,B88),B47))),((B35,(B66,B67)),B19))),((((B85,B86),(B13,((B80,B81),(B37,B38)))),((B23,B24),((B89,B90),B70))),(((B33,(B78,B79)),B29),(B65,(B98,B99)))))),((((B15,B16),((B68,B69),(B28,((B50,B51),B34)))),(((B31,B32),(B18,(B41,(B56,B57)))),(((B44,(B76,B77)),(B20,B21)),(B54,B55)))),((B48,B49),B10))),(B2,(B8,(B62,(B71,B72))))),((B58,B59),(B30,(B42,B43)))))));
