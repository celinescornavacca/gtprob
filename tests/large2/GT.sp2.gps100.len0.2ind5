((((((((A39,A40),(A70,A71)),(A18,(A31,A32))),((A6,(A74,A75)),((A7,(A19,((A24,(A59,A60)),A20))),A3))),(((((((A87,A88),(A68,A69)),A11),(A28,(A53,(A85,A86)))),A0),(((((A21,((A76,A77),(A64,A65))),(A16,(((A90,A91),(A62,A63)),(((A72,A73),A61),A49)))),(((A12,(A52,((A94,A95),A84))),(A10,(A78,A79))),((A80,A81),A5))),(((A47,A48),A4),((A37,(A92,A93)),A8))),((A9,(A14,A15)),(((A50,A51),(A33,A34)),(A25,A26))))),(((A35,A36),A29),(A23,(A82,A83))))),(A56,A57)),(((A27,(A66,A67)),((A17,(A54,A55)),(((A98,A99),((A45,A46),A38)),A13))),(A1,((((A43,A44),A42),A41),(((A30,(A58,(A89,(A96,A97)))),A22),A2))))),(((((B32,(B66,B67)),(B12,((B41,(B79,B80)),(((B29,(B49,(B51,B52))),B28),(B35,(B85,B86)))))),((B8,((B26,B27),(B33,B34))),(((B75,B76),B63),(B39,(B55,B56))))),((((B37,(B59,B60)),(B96,B97)),B18),B2)),((((B14,((B83,B84),(B68,B69))),((B71,B72),B45)),((((B0,((((B61,B62),B54),((B64,B65),B38)),B1)),((B13,(B20,B21)),B4)),(B57,B58)),((((B6,(B50,(B88,B89))),(B3,((B73,B74),B7))),(B25,((B77,B78),(B46,B47)))),(((B30,B31),(B11,(((B24,(B42,B43)),B15),(B70,(B81,B82))))),(B5,((B87,(B92,B93)),(B48,(B90,B91)))))))),((B44,(B94,B95)),(((B17,((((B98,B99),B53),B40),B36)),(B16,(B19,(B22,B23)))),(B9,B10))))));
