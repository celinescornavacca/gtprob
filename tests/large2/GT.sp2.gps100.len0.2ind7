(((A38,A39),((((A8,((A86,A87),((A72,A73),A25))),((A82,A83),((A15,(A19,A20)),(A92,A93)))),(((A90,A91),(A60,A61)),A21)),(((((A37,(A58,A59)),((A45,A46),(A32,(A50,A51)))),(A23,A24)),((((A49,(A80,A81)),(A78,A79)),(((A17,A18),A14),A4)),(((((A5,(A76,A77)),((A16,(((A56,A57),A40),(A52,A53))),A9)),(((A74,A75),(((A35,A36),(A41,A42)),(A84,A85))),(A29,A30))),A0),(((A12,(A63,(A69,(A98,A99)))),A6),(A1,(A2,(A13,(A94,A95)))))))),(((A26,((A67,A68),A62)),((A10,A11),((((A33,A34),(A64,(A70,A71))),(((A47,A48),((A31,(A54,A55)),A27)),((((A65,A66),A28),A22),(A88,A89)))),A3))),((A7,(A43,A44)),(A96,A97)))))),(((((B1,((((B79,B80),B9),(B46,(B52,B53))),((B83,B84),((B31,(B94,B95)),B11)))),(((B15,(B40,(B89,B90))),((B7,((B8,(B71,B72)),(B77,B78))),((B37,(B44,B45)),(B35,(B64,B65))))),(((((B81,B82),B6),(B18,B19)),B4),(B10,(B34,(B47,(B87,B88))))))),((B16,(B17,((B67,B68),B41))),((B48,B49),((B20,B21),(B50,B51))))),((B3,(B5,(((B85,B86),(((B14,(B56,B57)),(B12,((B24,((B62,B63),B59)),B13))),(B36,(B75,B76)))),((B54,B55),B38)))),(B0,((((B42,B43),(B58,(B69,B70))),(B32,B33)),(((B29,B30),(B39,(B66,(B73,B74)))),(((B91,(B98,B99)),((B26,B27),(B22,B23))),(B25,(B28,(B92,B93))))))))),(B2,((B60,B61),(B96,B97)))));
