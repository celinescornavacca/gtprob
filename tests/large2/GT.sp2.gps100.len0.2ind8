((((((A18,(A19,A20)),((A88,A89),(A27,A28))),A3),(((A44,(A46,(A92,A93))),A9),(((((A77,A78),(A54,(A67,A68))),(A60,(A63,A64))),A29),A8))),((((((A10,A11),((A12,(((A61,A62),(A75,A76)),A25)),(A34,A35))),A2),((A4,(((A90,A91),A55),A42)),((((A73,A74),A43),(A50,A51)),(A13,(((A58,(A94,A95)),A30),(A47,(A48,A49))))))),((((((A98,A99),(A21,(A37,A38))),(((A79,(A80,A81)),A36),((A40,A41),((A96,A97),A59)))),(((A45,(A71,A72)),(A52,A53)),(A32,A33))),(A7,((A17,((A65,A66),((A86,A87),A31))),A15))),((((A5,(((A56,A57),A26),(A6,(A14,(A16,(A23,A24)))))),A1),(A84,A85)),(((A39,(A69,A70)),A22),(A82,A83))))),A0)),(((((B5,((B46,(B73,B74)),(B79,B80))),(((B10,(B42,(B77,B78))),((((B35,B36),B8),B7),B2)),(((B86,B87),B85),B9))),((B22,B23),(B17,B18))),((B1,((B16,((B67,B68),B32)),(B20,B21))),(B19,(B25,B26)))),((((((B6,(B40,(B43,B44))),((B81,B82),B49)),((((((B92,B93),(B50,B51)),B41),B34),B13),(B14,(B28,B29)))),(B0,(B3,(B33,(B65,B66))))),(((((B88,B89),((B56,B57),B31)),(((((B69,B70),B64),B55),B52),B30)),B15),((B58,B59),((B94,B95),(B62,B63))))),((((B38,B39),((B45,(B90,B91)),B27)),((B71,B72),B61)),(((((B37,(B47,B48)),B4),(((B96,B97),(B24,(B60,(B98,B99)))),(B53,B54))),((B83,B84),(B75,B76))),(B11,B12))))));
