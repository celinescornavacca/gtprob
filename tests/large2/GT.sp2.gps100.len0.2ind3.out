-824.084869301
